src/tblib/pickling_support.py

```diff
@@ -22,14 +22,20 @@ def pickle_traceback(tb):
     return unpickle_traceback, (Frame(tb.tb_frame), tb.tb_lineno, tb.tb_next and Traceback(tb.tb_next))
 
 
-def unpickle_exception(func, args, cause, tb):
+# unpickle_exception_3_11() / pickle_exception_3_11() are used in Python 3.11 and newer
+
+def unpickle_exception_3_11(func, args, cause, context, notes, suppress_context, tb):
```

Comment:
What about https://pypi.org/project/exceptiongroup/ on older versions?

Reply:
I thiiink ExceptionGroup is orthogonal to `__context__`, `__notes__`, etc.?

The only ExceptionGroup-related functionality is in `_install_for_instance()`, and in that case I checked for the existence of an `exceptions` attribute, which should work with https://pypi.org/project/exceptiongroup/.